### DISCUSSION TITLE
Feature:  Fix Client 3.3.5a with patch ".MPQ" Files

### DISCUSSION THREAD

**Discussion by The-GhostRider** [azerothcore/azerothcore-wotlk#4462] [Ideas]
[patch-5.zip](https://github.com/azerothcore/azerothcore-wotlk/files/5936262/patch-5.zip)

**Describe your feature request or suggestion in detail**

In my opinion it would be right to Fix the "eventual" bugs related to the Game Client, as it could present some "Visual" bugs (such as spell descriptions and "Altered" Spell calculations).
As I have demonstrated in this PR: https://github.com/azerothcore/azerothcore-wotlk/pull/4408.

Although I had fixed the Spell: "Life Tap" from both CORE and DB (The spell "Mechanically" worked), but on the Spell Tooltip it wrote something else, and the Player could report a non-existent error or some of us could fix that same spell on an "Indicative" basis given by the Client which in the end proved to be false..

**Describe a possible solution to your Feature or Suggestion in detail**

In my opinion, the possible solution is to make a Section on Github for the Client Side Fixs.
And:
A: Load the first one (e.g :) patch-5.mpq
Later (in the same post)
B: (He noticed another bug in the game client)
B: Download the "patch-5.mpq" file from (A )'s post
B: Add his fix to "patch-5.mpq"
B: Update the "patch-5.mpq" file with a post under the post of (A) with the date of the update done
subsequently: (C) will be able to repeat the operation that did (B) to update his file.
In this way (X) can always have the latest most fixed update.

**Additional context**

Example of how to update an .MPQ file: https://youtu.be/x07nRtewCWM

[EDIT]
I added the "patch-5.mpq" attached to this post.
With this patch the Life Tap is fixed in all Ranks, just add the file in the "Data" folder of the Game.

P.S.
And I want to clarify, that this is not a "MOD", it is a "Repair".
Because this was a CODE ERROR, or rather! "A CODE UPDATE ERROR", they haven't updated the code in the client, they just forgot it..
So this update makes the Client a Super-BlizzLike! "BlizzShouldLike"

**Comment by Kitzunu** [comment]
Anything is possible, I just want to note that
1. You can never force someone to use a non-blizzlike client
2. This would need to be done in another repo as it is irelevant to the core imo

**Comment by Voxstrasza** [comment]
this isnt necessary because blizzard didn't fix the life tap tooltip until cataclysm; it's considered 'blizzlike'
